src/commands/hey.ts

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
(Same here - I won't comment it on each one of them, but you understand the idea - try to change the `module.exports` into `export` and `require` to `import`! As I mentioned earlier, you will _probably_ change the way you declare stuff in here anyways! We'll see as soon as you're there 😉)